openpype/hosts/houdini/api/plugin.py

```diff
@@ -168,6 +168,7 @@ class HoudiniCreator(NewCreator, HoudiniCreatorBase):
     """Base class for most of the Houdini creator plugins."""
     selected_nodes = []
     settings_name = None
+    _add_self_publish_button = False
 
```

Comment:
Is there a reason we prefix this with underscore but usually do not do so for similar attributes that get set from studio settings?

Reply:
Also if you wouldn't prefix it I believe you don't need the changes in apply settings just for this attribute since then it matches by name already

Reply:
I added that `_` prefix because I found out that I picked the the same name for the attribute 
and I wasn't comfortable with something like this 

```
if self.add_self_publish_button:
    add_self_publish_button(instance_node)
```

I feel a little dumb, why didn't I just pick another name for the attribute :) ?!